packages/core/src/external/commonwell/document/document-downloader-local.ts

```diff
@@ -238,7 +238,8 @@ export class DocumentDownloaderLocal extends DocumentDownloader {
         this.config.capture.error(error, { extra: { ...additionalInfo, error } });
 
       if (error instanceof CommonwellError && error.cause?.response?.status === 404) {
-        throw new MetriportError("CW - Document not found", error, additionalInfo);
+        console.log(`CW - Document not found, error: ${error}, additionalInfo: ${additionalInfo}`);
+        return;
```

Comment:
Wouldn't it be better to throw `NotFoundError` and let the caller decide whether to fail silently? 

If we don't throw here, the calling code will continue to execute as if the document had downloaded successfuly.

Reply:
Yes, that's way better